web/src/lib/components/admin-page/settings/backup-settings/backup-settings.svelte

```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+  import type { SystemConfigDto } from '@immich/sdk';
+  import { isEqual } from 'lodash-es';
+  import { fade } from 'svelte/transition';
+  import type { SettingsResetEvent, SettingsSaveEvent } from '../admin-settings';
+  import SettingInputField, {
+    SettingInputFieldType,
+  } from '$lib/components/shared-components/settings/setting-input-field.svelte';
+  import SettingSwitch from '$lib/components/shared-components/settings/setting-switch.svelte';
+  import SettingButtonsRow from '$lib/components/shared-components/settings/setting-buttons-row.svelte';
+  import { t } from 'svelte-i18n';
+  import FormatMessage from '$lib/components/i18n/format-message.svelte';
+
+  export let savedConfig: SystemConfigDto;
+  export let defaultConfig: SystemConfigDto;
+  export let config: SystemConfigDto; // this is the config that is being edited
+  export let disabled = false;
+  export let onReset: SettingsResetEvent;
+  export let onSave: SettingsSaveEvent;
+
+  $: cronExpressionOptions = [
+    { title: $t('interval.night_at_midnight'), expression: '0 0 * * *' },
+    { title: $t('interval.night_at_twoam'), expression: '0 02 * * *' },
+    { title: $t('interval.day_at_onepm'), expression: '0 13 * * *' },
+    { title: $t('interval.hours', { values: { hours: 6 } }), expression: '0 */6 * * *' },
+  ];
+</script>
+
+<div>
+  <div in:fade={{ duration: 500 }}>
+    <form autocomplete="off" on:submit|preventDefault>
+      <div class="ml-4 mt-4 flex flex-col gap-4">
+        <SettingSwitch
+          title={$t('admin.backup_database_enable_description')}
+          {disabled}
+          bind:checked={config.backup.database.enabled}
+        />
+
+        <div class="flex flex-col my-2 dark:text-immich-dark-fg">
+          <label class="font-medium text-immich-primary dark:text-immich-dark-primary text-sm" for="expression-select">
+            {$t('admin.cron_expression_presets')}
+          </label>
+          <select
```

Comment:
There's also `<SettingSelect>`, not sure if you want to use that instead.

Reply:
Nice, wasn't aware of that, will make a separate PR to update this on the library settings too :)